components/layout/display_list/items.rs

```diff
@@ -550,13 +550,14 @@ impl ClippingRegion {
     pub fn translate(&self, delta: &LayoutVector2D) -> ClippingRegion {
         ClippingRegion {
             main: self.main.translate(delta),
-            complex: self.complex.iter().map(|complex| {
-                ComplexClipRegion {
+            complex: self.complex
+                .iter()
+                .map(|complex| ComplexClipRegion {
```

Comment:
I think this is uglier btw, but I assume this has been done with rustfmt... Oh well.